packages/itwinui-css/src/file-upload/file-upload.scss

```diff
@@ -69,8 +70,120 @@
     visibility: visible;
 
     > svg,
-    .iui-template-icon {
+    .iui-template-icon,
+    .iui-file-card-icon,
+    .iui-file-card-empty-icon {
       fill: var(--iui-color-icon-accent);
     }
   }
 }
+
+@mixin iui-file-card {
+  @include iui-reset;
+  width: 100%;
+  line-height: normal;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: calc(var(--iui-size-s) * 0.5) var(--iui-size-s);
+  border-radius: var(--iui-border-radius-1);
+
+  &:focus-within {
+    outline: 2px solid var(--iui-color-text-accent);
+    outline-offset: -2px;
+  }
+
+  > svg {
```

Comment:
is this needed? `iui-file-card-icon` should cover this case.

Reply:
This has been removed